<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Project Name:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="projectname">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>End date:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="startdate">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="enddate">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="descriptionText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createprojectButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editprojectButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteprojectButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>50</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshprojectButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>50</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="projectList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>541</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="storyList">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>541</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Projects</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stories</string>
    </property>
   </widget>
   <widget class="QLabel" name="Username_label">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Logged in as:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1166</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
